sdks/typescript/client-typescript-eventsource/README.md

```diff
@@ -1,20 +1,20 @@
 # Javascript/Typescript Client SDK for FeatureHub
 
 Welcome to the Javascript/Typescript SDK implementation for FeatureHub. It is the supported version, but it does not mean
-you cannot write your own, please follow Contribution guidelines.
+you cannot write your own, please follow [Contributing](https://github.com/featurehub-io/featurehub/blob/master/.github/CONTRIBUTING.adoc) guidelines.
 
 ## Overview
 
-Below explains how to implement FeatureHub SDK using Javascript/Typescript for applications, like node.js backend servers or Web front-ends (e.g. React). 
+Below explains how to implement FeatureHub SDK using Javascript/Typescript for applications like Node.js backend server or Web front-end (e.g. React). 
```

Comment:
Below explains how you can use the FeatureHub SDK in Javascript or Typescript for applications like